Plugins/Mute/DataManager.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Data.Models.Client;
+using SharedLibraryCore.Interfaces;
+using static System.Enum;
+
+namespace Mute;
+
+public class DataManager
+{
+    public DataManager(IMetaServiceV2 metaService)
+    {
+        _metaService = metaService;
+    }
+
+    private readonly IMetaServiceV2 _metaService;
+
+    public async Task<MuteState> ReadPersistentData(EFClient client)
+    {
+        var clientMeta = (await _metaService.GetPersistentMeta(Plugin.MuteKey, client.ClientId))?.Value ??
```

Comment:
You should check first to see if it's in the additional  properties so we don't have to make another trip to the database. 
![image](https://user-images.githubusercontent.com/6732771/185760973-2c6ca0ce-bf1d-4869-b7ae-f886311a4d93.png)
The most up to date value should be set when they join, so check the additional properties first, and if it's null then go to the database.